ledger-core/virtual/execute/execute.go

```diff
@@ -327,7 +327,10 @@ func (s *SMExecute) stepDeduplicate(ctx smachine.ExecutionContext) smachine.Stat
 
 	// deduplication algorithm
 	action := func(state *object.SharedState) {
-		deduplicateAction = s.deduplicate(state, msg)
+		deduplicateAction0, msg0 := s.deduplicate(state)
```

Comment:
why do you use variables with 0 postfixes?

Reply:
What better name for this case?